### PR TITLE
Feature/spanish

### PR DESCRIPTION
**Pull request by jehernandezrodriguez**
Creation of a Spanish version for the files. Spanish language was added in the i18next configuration.

I upload key translations (spanish), I don't know if it's necessary.

### CODE REVIEW

File: app/pages/patientdata/patientdata.js [hunk 1]
```diff
@@ -192,7 +194,7 @@ export let PatientData = translate()(React.createClass({
     return this.renderChart();
   },
 
-  renderEmptyHeader: function(title = 'Preparing Chart Data') {
+  renderEmptyHeader: function(title = t('Preparing Chart Data')) {
```
Comment (cbwebdevelopment) [inline app/pages/patientdata/patientdata.js:197]: Using `t` in the function signature here solves the issue you were addressing, but is causing the automated `lint` test to fail.

I suggest setting the default title within the function body.  Something like:
```js
  renderEmptyHeader: function(title) {
    const { t } = this.props;
    const headerTitle = title || t('Preparing Chart Data');
    return (
      <Header
        chartType={'no-data'}
        inTransition={false}
        atMostRecent={false}
        title={headerTitle}
        ref="header"
      />
    );
  },
```

File: i18next-parser.config.json [hunk 2]
```diff
@@ -5,5 +5,5 @@
     "js": ["JavascriptLexer", "JsxLexer"]
   },
   "sort": true,
-  "locales": ["en", "fr"]
+  "locales": ["en", "fr","es"]
```
Comment (cbwebdevelopment) [inline i18next-parser.config.json:8]: Tiny nitpick -- should have a space after the comma for consistency.

**Comment by cbwebdevelopment** [comment]
One other thing: we'll need you to agree to our VCLA with a comment here before I can merge as well.

The details can be found at https://developer.tidepool.org/contributors/#making-contributions

**Comment by jehernandezrodriguez** [comment]
I agree to the terms of Tidepool Project’s Volunteer/Contributor License Agreement v1.1 as it exists at http://tidepool-org.github.io/files/TidepoolVCLA-1.1.pdf on 06/april/2020.

**Pull request merged**